source/dynamical_systems/CMakeLists.txt

```
@@ -51,3 +51,13 @@ if (BUILD_TESTING)
   )
   add_test(NAME test_${LIBRARY_NAME} COMMAND test_${LIBRARY_NAME})
 endif ()
+
+if(${PKG_CONFIG_FOUND})
+  set(PKG_NAME ${LIBRARY_NAME})
+  set(PKG_DESC "This library provides a set of classes to represent dynamical systems; functions which map a state to a state derivative.")
```

The semicolon here is not quite right - I would suggest a hyphen or colon instead.
```suggestion
  set(PKG_DESC "This library provides a set of classes to represent dynamical systems: functions which map a state to a state derivative.")
```